plugin/src/main/java/jenkins/plugins/openstack/compute/UserDataVariableResolver.java

```diff
@@ -77,6 +80,21 @@
                 "Labels of the node.",
                 r -> r.labelString
         );
+        stub(
+                "AGENT_NAME",
+                "Agent name",
+                r -> r.serverName
+        );
+        stub(
+                "JENKINS_INSTANCE_IDENTITY",
```

Comment:
Question: What is the use-case for accessing instance identity?

Reply:
Please look on:
https://github.com/jenkinsci/remoting/blob/master/docs/inbound-agent.md#connect-directly-to-tcp-port
It is one of arguments.

Reply:
Thanks. Makes sense.